modules/fs2/src/main/scala/com/snowplowanalytics/snowplow/enrich/fs2/io/Source.scala

```diff
@@ -30,6 +30,26 @@ import java.nio.file.{Files, Path}
 
 object Source {
 
+  /**
+   * Number of threads used internally by permutive library to handle incoming messages.
+   * These threads do very little "work" apart from writing the message to a concurrent Queue.
+   * Overrides the permutive library default of `3`.
+   */
+  val ParallelPullCount = 1
+
+  /**
+   * Configures the "max outstanding element count" of pubSub.
+   *
+   * This is the principal way we control concurrency in the app; it puts an upper bound on the number
+   * of events in memory at once. An event counts towards this limit starting from when it received
+   * by the permutive library, until we ack it (after publishing to output). The value must be large
+   * enough that it does not cause the sink to block whilst it is waiting for a batch to be
+   * completed.
+   *
+   * (`1000` is the permutive library default; we re-define it here to be explicit)
+   */
+  val MaxQueueSize = 1000
```

Comment:
> This is the principal way we control concurrency in the app

Would be great to make it configurable so that a small machine and a big one can use different concurrency